index.js

```diff
@@ -0,0 +1,124 @@
+$(document).ready(() => {
+  const baseUrl = "https://trektravel.herokuapp.com";
+
+  const travelList = function travelList() {
+    const listPartial = "/trips";
+    if (".individual-trip-section") {
+      $(".individual-trip-section").remove();
+    }
+    if ("#ajax-results p") {
+      $("#ajax-results p").remove();
+    }
+    $.get(baseUrl + listPartial, response => {
+      $(".list-trips").html(
+        `<div class = "removable-trip-list  row small-up-2 medium-up-2 large-up-3 "></div>`
```

Comment:
This success callback and the one for fetching a single trip are just complex enough that it would probably be worthwhile to extract them as a named function, defined at the top of the file.